src/org/usfirst/frc/team694/robot/commands/LiftDownCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class LiftDownCommand extends Command {
+
+    public LiftDownCommand() {
+        requires(Robot.lift);
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.goDown();
```

Comment:
If isFinished returns true, execute never runs. Move this elsewhere

Pro Tip: For commands that only run once, look at [InstantCommand](http://first.wpi.edu/FRC/roborio/release/docs/java/)

Reply:
Also, you don't disable your Lift's break before running it. Take a look at your other command where you call Robot.lift.setBrakeOff()